src/app/scripts/cac/map/cac-map-templates.js

```diff
@@ -320,7 +325,9 @@ CAC.Map.Templates = (function (Handlebars, moment, Utils) {
         });
 
         Handlebars.registerHelper('datetime', function(dateTime) {
-            return new Handlebars.SafeString(new Date(dateTime).toLocaleTimeString());
+            // round to the nearest minute
```

Comment:
Is this comment correct? Isn't the string below essentially taking the ceiling of the current minute?